_posts/2020-06-03-dalalyan20a.md

```diff
@@ -23,12 +23,12 @@ lastpage: 1341
 page: 1331-1341
 order: 1331
 cycles: false
-bibtex_author: Dalalyan, Arnak and Schreuder, Nicolas and Brunel, Victor-Emmanuel
+bibtex_author: Schreuder, Nicolas and Dalalyan, Arnak and Brunel, Victor-Emmanuel
```

Comment:
This order doesn't match the PDF ... where Dalalyan seems to be the last author.